frontend/degree-plan/components/Dock/Dock.tsx

```diff
@@ -183,6 +200,12 @@ const Dock = ({ user, login, logout, activeDegreeplanId  }: DockProps) => {
                             {dockedCourses.map((course) => 
                                 <AnimatedDockedCourseItem course={course} isDisabled={false} />
                             )}
+                            <CloseIcon>
+                                <CourseXButton onClick={(e) => {
+                                    temps.forEach((course) => handleRemoveCourse(course.full_code));
```

Comment:
Whoops I meant `dockedCourses` but doesn't matter anyway since this isn't gonna be merged